ci/vendor/pipeline-fragments.lib.yml

```diff
@@ -335,7 +335,7 @@ plan:
         BOT_BRANCH: #@ data.values.git_charts_bot_branch
         CHARTS_SUBDIR: #@ data.values.gh_repository
       run:
-        path: pipeline-tasks/ci/vendor/tasks/open-charts-pr.sh
+        path: pipeline-tasks/ci/vendor/tasks/chart-open-charts-pr.sh
```

Comment:
What does this have a `chart-` prefix? Its run in the context of the app pipeline right?